docs/components/plugins/api/APISectionUtils.tsx

```diff
@@ -95,6 +96,8 @@ export const resolveTypeName = ({
         t.type === 'array' ? `${t.elementType?.name}[]` : `${t.name || t.value}`
       )
       .join(' | ');
+  } else if (declaration && declaration.signatures) {
```

Comment:
```suggestion
  } else if (declaration?.signatures) {
```
I believe it will work too 😉